tools/update_goversion/README.md

```diff
@@ -0,0 +1,8 @@
+# Update Go Version
+
+`go run tools/update_goversion/main.go $LATEST_GO_VERSION`
```

Comment:
Can we give an example, we seem to have an inconsistent use of prefixing with `v` or not.
Looks like here we do not need the `v` prefix